BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-132: Change password employee

- Add modal to change password
- Add password input and repeat password input
- Add validations to inputs
- Add success or error modal message

QA:
How to test?
Log in as employee (try not to log as the employee with the email: flune19@desdev.cn, check de database for more employees data [Database: employees](https://cloud.mongodb.com/v2/6346f3df659ee31f0aaca3d1#metrics/replicaSet/63470669f3e4223d9265680f/explorer/test/employees/find)).
Go to the "Profile" page.
When pressing the "Change password" button, a modal with two inputs should appear.
Pressing the "cancel" button, or the "X" button should close the modal.
When pressing the "Confirm" button you should be submitting the form, this modal will close, and a confirmation modal will appear, if you confirm, a message modal will say if the request was succesfully or not.
If the inputs aren't completed when you try to submit the form, validations won't be passed, so you shoul see red messages, and the modal will be still open.
Try changing the password succesfully on one employee, then logout, and confirm that the password has changed re login with the new password.
